### PR TITLE
Butterdogs

### PR DESCRIPTION
**Pull request by ninjanomnom**
:cl: ninjanomnom
add: You can now make a new tasty traditional treat: butterdogs. Watch out, they're slippery.
/:cl:

image

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
image
sum ting wong?

**Comment by GunHog** [comment]
### Exotic butters!!

**Comment by imsxz** [comment]
why though

**Comment by ninjanomnom** [comment]
the bot apparently thinks outdated icon files means removal

**Comment by RemieRichards** [comment]
The bot is correct.
In the diff of CURRENT FILE -> YOUR FILE those icons have disappeared, they have been removed.

The only way to fix this would be to compare file revisions directly, which is too much of a pain in the ass to be worth it, I'd imagine.

**Comment by WJohn** [comment]
it's... it's a block of butter in a hotdog bun?

sure. why not.

**Comment by The-clown-known-as-Pepe** [comment]
homer's moon waffles when?

image

**Comment by RandomMarine** [comment]
When I saw the PR title I thought it would be a dog variant of the cak

**Comment by arsserpentarium** [comment]
Why ratkebab was removed?

**Comment by praisenarsie** [comment]
wasn't removed, he just conflicted because his master was out of date.

**Comment by fludd12** [comment]
Should make it taste disgusting unless the person is fat

File: code/modules/food_and_drinks/food/snacks_bread.dm [hunk 1]
```diff
@@ -238,4 +238,19 @@
...
+	icon_state = "butterdog"
+	bitesize = 1
+	filling_color = "#F1F49A"
+	list_reagents = list("nutriment" = 5)
+	bonus_reagents = list("nutriment" = 1, "vitamin" = 1)
+	tastes = list("butter", "exotic butter")
+
+/obj/item/reagent_containers/food/snacks/butterdog/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/slippery, 80)
```
Comment (kevinz000) [inline code/modules/food_and_drinks/food/snacks_bread.dm]: griefy...

**Comment by Niknakflak** [comment]
I was hoping this would be something like pugs and corgis made out of butter in relation to the cak and was severely dissapointed

**Comment by Cyberboss** [comment]
I agree with @Qbopper

**Comment by PKPenguin321** [comment]
🤢

**Comment by banthebantz** [comment]
🤢

**Comment by darkpaladin109** [comment]
🤢

**Comment by AnturK** [comment]
Meme foods cannot be stopped.

**Pull request merged**